README.md

```diff
@@ -144,14 +147,15 @@ your/favorite/dir/
 For `yarn start www`, see note about [CORS](#cors).
 
 The `start` CLI commands provide some configuration options and, in general, the default values of these options are set to environment variables. For example:
+
 - to change where site content should be stored, alter `COURSE_CONTENT_PATH` and `WWW_CONTENT_PATH` environment variables. (Can be overriden with `--content-dir` cli option.)
-- to change the github org from which site content is fetched alter `GIT_CONTENT_SOURCE` environment variable (can be override with --git-content-source). *By default content is pulled from rc.
+- to change the github org from which site content is fetched alter `GIT_CONTENT_SOURCE` environment variable (can be override with --git-content-source). \*By default content is pulled from rc.
```

Comment:
```suggestion
- to change the github org from which site content is fetched alter `GIT_CONTENT_SOURCE` environment variable (can be override with --git-content-source). (*By default content is pulled from rc*).
```
I suspect this was meant as a parenthetical statement, not an actual asterisk. Could be wrong :shrug: